shared/js/activeRoomStore.js

```diff
@@ -968,6 +996,7 @@ loop.store.ActiveRoomStore = (function() {
       loop.request("AddBrowserSharingListener", this.getStoreState().windowId)
         .then(this._browserSharingListener);
       loop.subscribe("BrowserSwitch", this._browserSharingListener);
+      loop.subscribe("CursorPosition", this._sendCursorPositionListener);
```

Comment:
as in comment https://github.com/mozilla/loop/pull/90/files#r50117129, `CursorPositionChange`